src/controllers/serverRoomIDController.ts

```diff
@@ -111,24 +111,48 @@ export const createRoom = async (req: Request, res: Response) => {
 // Get all rooms
 export const getAllRooms = async (req: Request, res: Response) => {
   try {
-    const rooms = await RoomId.find({});
+    const { query } = req.query;
```

Comment:
instead of query in params use search variable

Reply:
ok

Reply:
should i use like this:-  const { param } = req.param;